composer.json

```diff
@@ -7,6 +7,7 @@
         "dpc-sdp/tide_core": "^1.5.4",
         "dpc-sdp/tide_media": "^1.5.1",
         "dpc-sdp/tide_webform": "^1.1.8",
+        "dpc-sdp/tide_site": "^1.3.2",
```

Comment:
@vincent-gao I don't think Tide Grant should have a dependency on Tide Site. Rather Tide Grant should check if Tide Site is enabled before executing anything that relies on it.

Reply:
Thanks @GROwen .

Reply:
Hi @GROwen , could you please CR it again.